src/web/app.py

```diff
@@ -0,0 +1,44 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+import os
+import openai
+from flask import Flask, render_template, request
+from pathlib import Path
+from dotenv import load_dotenv
+from azure.identity import DefaultAzureCredential
+
+default_credential = DefaultAzureCredential()
+token = default_credential.get_token("https://cognitiveservices.azure.com/.default")
```

Comment:
I believe the token will expire in ~24 hours. We might want to move this line to the completions_demo() function and issue a new token every call, or refresh the token ourselves when it expires.

Reply:
I think as-is is fine for a quick sample.